helm/conjur-app-deploy/charts/app-secrets-provider-init/templates/conjur_authn_configmap.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
```

Comment:
Should this ConfigMap manifest be conditionalized on `.Values.conjur.authnConfigMap.create` being set to `true`?

Reply:
Done